pkg/registry/scripts/projects.js

```diff
@@ -152,5 +197,96 @@
                 return methods.create(project);
             };
         }
+    ])
+
+    .controller('UserNewCtrl', [
+        '$q',
+        '$scope',
+        "kubeMethods",
+        function($q, $scope, methods) {
+            var fields = {
+                name: "",
+                password: ""
```

Comment:
I think this dialog isn't supposed to have a password field. Should it be an identity field instead?